### ISSUE TITLE
[Suggestion] Validation error messages

### ISSUE THREAD

**Issue by romanoff** [gavinwahl/postgres-json-schema#4]
My understanding is that at the moment this extension allows you to get answer if validation failed or succeeded. But would be great if you could also error messages. Possible format:
```
{"name": ["is required"]}
```
Not sure about exact format, but that would definitely be useful in some cases (my use-case is trying to process http request in postgres function. And this would provide validation messages )

**Comment by gavinwahl** [comment]
I agree, it would be nice to have this. My application doesn't need it because I validate in the application as well, and the check constraint is just for integrity.

It seems pretty hard to implement so I didn't do it, but I would accept a patch that did.
